### DISCUSSION TITLE
Use of URL Switch

### DISCUSSION THREAD

**Discussion by dbrighton** [CycloneDX/cyclonedx-node-module#311] [Ideas]
I need to use the URL switch to use our companay's private Artifactroy.  How do pass login credentials?

**Comment by jkowalleck** [comment]
Thanks for the great issue report, @dbrighton .
I might have some questions, still: 

* What is "the URL switch"?
* Why using it? 
* What are trying to achieve? 
* What alternatives did you try/consider?

**Comment by jkowalleck** [comment]
this conversation got closed due to #321
